NSrtm.Core/FileExtensions/Pgm/PathResolvers/PgmPathResolverZip.cs

```diff
@@ -0,0 +1,17 @@
+﻿using JetBrains.Annotations;
+
+namespace NSrtm.Core
+{
+    internal class PgmPathResolverZip : PgmPathResolverCaching
```

Comment:
Yes, but i don't use gz format AFAIK.
At the moment i use raw files. In the future i want to operate on tar.bz2 and zip files